fedimint-core/src/task.rs

```diff
@@ -172,11 +172,7 @@ impl TaskGroup {
         Fut: Future<Output = R> + MaybeSend + 'static,
         R: MaybeSend + 'static,
     {
-        use tracing::{info_span, Instrument, Span};
-
         let name = name.into();
-        // new child span of current span
-        let span = info_span!(parent: Span::current(),"task", name);
```

Comment:
`parent` leads to a lot of not very useful nested spawns, and the whole thing is redudant with with the `runtime::spawn` doing the same thing.